bin/history+

```diff
@@ -36,6 +36,9 @@ function __history+() {
                 echo
                 echo "TEXT is a string searched in the history+ database."
                 echo
+                echo "All parameters can be specified in any order. If TEXT is"
```

Comment:
I suggest removing these lines since they are just noise in this case. It would make sense if we accept several parameters in some complex way, but it's not the case here. The usage is pretty obvious.